README.md

```diff
@@ -1,31 +1,46 @@
+If you have worked with an ApostropheCMS project previously, the backend repo should look as expected. There are a number of custom modules, providing new pages, pieces, and widgets, located in the `modules` folder. There are also a number of modules that are improving the core Apostrophe modules located in the `modules/@apostrophecms` folder. For a full understanding of Apostrophe you should consult the [documentation](https://docs.apostrophecms.org/), but we will touch on a few highlights later in this document.
 
 Like any ApostropheCMS project, after creating a new module it needs to be registered in the `app.js` file. For pages or 'piece-type-pages' they also need to be registered in the `modules/@apostrophecms/page/index.js` file.
 
 The majority of [module configuration settings](https://docs.apostrophecms.org/reference/module-api/module-overview.html#module-configuration) will continue to operate as normal since they are involved in configuring the behavior and functionality of the Admin UI, request routing, which is still being handled by the ApostropheCMS backend server, or interaction with MongoDB.
 
-### Important differences
+### Important ApostropheCMS differences
 Where this project differs from a normal ApostropheCMS project is that no frontend code should be included in your modules. So, client-side JavaScript and styling, normally added to the `modules/custom-module/ui/src` folder will now be included in the Astro project. This also includes templates in the `views` folder of most modules. One exception is the `modules/@apostrophecms/home-page` module. This module provides a "fall-back" for users who navigate to the Apostrophe server (located by default at `localhost:3000` during development) and simply loads the core `views/layout.html` file. This core file has been modified in this project to provide info about the project status and not load any of the ApostropheCMS Admin UI.
 
 Equally, certain [module customization functions](https://docs.apostrophecms.org/reference/module-api/module-overview.html#customization-functions) that deal with front-end functionality should not be used. This includes the `helper()` and `extendHelpers()` functions for providing Nunjucks template helpers, the `components()` method that provide asynchronous template components, and the `renderRoutes()` function to return a rendered template.
 
+An advantage of using Astro as a frontend unlike an ApostrophCMS-only project, is that the templates for widgets can be used as templates outside specialized `area` schema fields. While this removes the ability to edit the widgets in-context, it does provide flexibility for page design without code repetition. However, this means that the widget schema needs to be added to the schema fields of the page. To facilitate this, some of the widget schema fields have been moved to the `lib/schema-mixins` folder. This allows them to be imported and used in both the main widget module and in the home page template.
```

Comment:
ApostrophCMS -> ApostropheCMS

Also I might start with "Unlike an ApostropheCMS-only project, ..."

Cool technique

Reply:
Similar to writing an async component or fragment used from both a page template and a widget template

Reply:
Changed.